.cbmc-batch/source/cbmc_invariants.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cbmc_invariants.h>
+
+#include <ec_utils.h>
+#include <evp_utils.h>
+
+#include <cipher_openssl.h>
+
+bool aws_cryptosdk_md_context_is_valid_cbmc(struct aws_cryptosdk_md_context *md_context) {
+    return aws_cryptosdk_md_context_is_valid(md_context) && evp_md_ctx_is_valid(md_context->evp_md_ctx);
```

Comment:
It can't be checked from SDK code, the is_valid part is all that can be checked in the SDK.  There needs to be some discussion of this.   Explain that _cbmc is the naming convention making this distinction.

Reply:
Added comment to the file clarifying this.